internal/provider/common/staticsite.go

```diff
@@ -15,6 +16,7 @@ type WrappedStaticSite struct {
 	EnvVars              *[]client.EnvVarWithCursor
 	Headers              *[]client.Header
 	NotificationOverride *notifications.NotificationOverride
+	LogStreamOverride    *logs.ResourceLogStreamSetting
```

Comment:
I don't believe log stream overrides are supported for static sites, since they don't have logs.